### ISSUE TITLE
Allow to add multiple routes to private LAN of Kubernetes node pool

### ISSUE THREAD

**Issue by Programie** [ionos-cloud/ionosctl#273]
### Use-cases

As of now, it's only possible to add a single route network + gateway IP per private LAN added to a Kubernetes node pool.

But it is possible to add multiple routes in the DCD.

### Attempted Solutions

```bash
ionosctl k8s nodepool lan add --cluster-id $CLUSTER_ID --nodepool-id $NODEPOOL_ID --lan-id 1 --network $NETWORK --gateway-ip $GATEWAY_IP
```

Specifying multiple networks in `$NETWORK` using spaces as separator results in only using the first subnet. Using something else like a comma as separator breaks the CIDR validation as the whole string is used as CIDR.

Specifying `--network` and `--gateway-ip` multiple times for each network-gateway combination also has no other effect than just using the first network.

### Proposal

Allow to specify multiple routes either by space/comma separating them or allowing to pass `--network` and `gateway-ip` multiple times.

The best solution might be to allow specifying the network and gateway together like `--network $NETWORK_1,$GATEWAY_1 --network $NETWORK_2,$GATEWAY_2`. But using something like `--network $NETWORK_1,$NETWORK_2 --gateway-ip $GATEWAY_1,$GATEWAY_2` would be also OK.

**Comment by avirtopeanu-ionos** [comment]
Hi, while the first solution is definitely more user friendly, I think we're gonna have to go with the second variant to keep backwards compatibility

**Comment by avirtopeanu-ionos** [comment]
Hello, thank you for this feature request. This has been implemented in [release v6.6.2](https://github.com/ionos-cloud/ionosctl/releases/tag/v6.6.2)

Have a nice day!
